far/farlang.templ.m4

```diff
@@ -13287,16 +13287,16 @@ MGetCodePageBottomShortTitle
 "Ctrl+H, Del, F4"
 
 MGetCodePageEditCodePageName
-"Изменить имя кодовой страницы"
-"Edit code page name"
-"Upravit název kódové stránky"
-"Codepagename bearbeiten"
-upd:"Edit code page name"
-upd:"Edit code page name"
-"Editar nombre de página de códigos"
-"Upraviť názov tabuľky znakov"
-"Modifica Nome Codepage"
-"Змінити ім'я кодової сторінки"
+"Переименование кодовой страницы"
+"Rename code page"
```

Comment:
`the code page` maybe?

Or just `Rename`, as in other action dialogs (Copy, Rename, Delete etc.)?

Reply:
I'd rather not. In the `Rename/Move` dialog, the edit control has the `Rename or move Foo,txt to:` caption. Here, the edit control has no caption at all, If the dialog caption also does not tell what to rename, the result may be confusing. Consider:
```
╔═══════════════════ Rename ═══════════════════╗
║ IBM EBCDIC - U.S./Canada                    ↓║
╟──────────────────────────────────────────────╢
║         { OK } [ Cancel ] [ Reset ]          ║
╚══════════════════════════════════════════════╝
```
If you like, I can make this dialog more like those for file operations. Better under a separate change. Please let me know.

Reply:
Indeed, it's a little terse - the edit control needs at least "Name:" caption.
We can handle that later.

Reply:
And I changed to `Rename the code page`. Sorry for missing it on the first round.